difference-of-squares/difference_of_squares.rb

```diff
@@ -0,0 +1,22 @@
+class Squares
+  def initialize(num)
+    @num = num
+    sum_of_num=0
+    sum_of_square=0
+    (0..@num).each do |i|
+        sum_of_num = sum_of_num + i
+        sum_of_square = sum_of_square + i*i
+      end
+      @sum_of_num=sum_of_num*sum_of_num
+      @sum_of_squares=sum_of_square
+  end
```

Comment:
Never perform any business logic in initialize method. Use of initialize method is for assignment and initialization of some vars